deliver/lib/deliver/runner.rb

```diff
@@ -78,7 +78,7 @@ def upload_binary
         )
       end
 
-      transporter = FastlaneCore::ItunesTransporter.new(options[:username])
+      transporter = FastlaneCore::ItunesTransporter.new(options[:username], nil, false, options[:itc_provider])
```

Comment:
The params being: `(user = nil, password = nil, use_shell_script = false, provider_short_name = nil)`